### PR TITLE
Styles to workspece projects page

### PR DESCRIPTION
**Pull request by ShevchukTania**
**STATUS**
READY

**DESCRIPTION**

**CHECK LIST**
- [x] specs added
- [x] specs passed
- [ ] code coverage >90%
- [x] DB indexes added if needed
- [x] seed.rb file is updated if needed
- [x] PR is reviewed manually again (to make sure you have 100% ready code)
- [x] PR doesn't have conflicts with main branch
- [x] I've checked new feature as logged in and logged out user if needed
- [x] "let!" is used in specs

image
image

### CODE REVIEW

File: app/assets/javascripts/account/workspaces.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+jQuery(document).ready(function () {
+  $('.modal-background').click(function(){
+    console.log('hej');
```
Comment (igorkasyanchuk) [inline app/assets/javascripts/account/workspaces.js]: remove console.log from code

File: app/assets/javascripts/account/workspaces.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+jQuery(document).ready(function () {
```
Comment (igorkasyanchuk) [inline app/assets/javascripts/account/workspaces.js]: use turbolinks callback, and understand why

File: app/assets/stylesheets/workspaces.scss [hunk 3]
```diff
@@ -10,3 +10,65 @@ span#new-member {
...
+      padding-left: 5px;
+    }
+  }
+}
+
+.column.is-one-fifth.members {
+  background-color: #f3f3f3;
+  border-radius: 10px;
+}
+.modal-background {
```
Comment (igorkasyanchuk) [inline app/assets/stylesheets/workspaces.scss]: move to "modal.scss" modal styles

File: app/views/account/workspaces/show.html.slim [hunk 4]
```diff
@@ -1,29 +1,41 @@
...
+                    | Add a project
+                .card-content
+                  = fa_icon 'plus 2x'
+                footer.card-footer
+      - else
+          .columns.is-multiline
+            .column.is-2
+              .card.empty
+                = link_to new_account_workspace_project_path(@workspace), remote: true do
+                  header.card-header
```
Comment (igorkasyanchuk) [inline app/views/account/workspaces/show.html.slim:35]: is it ok to add "header" element here? this is how bulma suggest to do it?

Comment (ShevchukTania) [reply app/views/account/workspaces/show.html.slim:35]: yes. It's bulma's markup

File: app/assets/stylesheets/projects.scss [hunk 5]
```diff
@@ -1,8 +1,26 @@
...
   font-size: 13px;
   text-align: center;
   border: 1px solid lightgrey;
-}
\ No newline at end of file
+  line-height: 24px;
+}
+
+.image {
+  display: inline-block;
```
Comment (igorkasyanchuk) [inline app/assets/stylesheets/projects.scss]: this is too big change

don't override bulma css classes

File: app/assets/stylesheets/workspaces.scss [hunk 6]
```diff
@@ -10,3 +10,65 @@ span#new-member {
   min-width: 200px;
   margin-right: 1rem;
 }
+
+.card {
+  background-color: #96eac3;
+  border-radius: 1em;
+    .image {
```
Comment (igorkasyanchuk) [inline app/assets/stylesheets/workspaces.scss]: alignment

File: app/assets/stylesheets/workspaces.scss [hunk 7]
```diff
@@ -10,3 +10,65 @@ span#new-member {
...
+}
+
+.column.is-one-fifth.members {
+  background-color: #f3f3f3;
+  border-radius: 10px;
+}
+.modal-background {
+  background-color: rgba(19, 19, 19, 0.56);
+}
+.modal-content {
```
Comment (igorkasyanchuk) [inline app/assets/stylesheets/workspaces.scss]: if you moved - why it's still here?

File: app/assets/stylesheets/workspaces.scss [hunk 8]
```diff
@@ -10,3 +10,65 @@ span#new-member {
   min-width: 200px;
   margin-right: 1rem;
 }
+
+.card {
```
Comment (igorkasyanchuk) [inline app/assets/stylesheets/workspaces.scss]: .card is also bulma class?
don't make this global overide 
use own classes

Comment (igorkasyanchuk) [reply app/assets/stylesheets/workspaces.scss]: because if we want to use "card" anywhere else we will have problems

**Review by igorkasyanchuk — APPROVED** [review]

**Pull request merged**